src/main/java/edu/mbrooks/advancedjava/interfaces/Icollectable.java

```diff
@@ -0,0 +1,25 @@
+/**
+ * Program Name:    Icollectable.java
+ * Author:          Michael Brooks
+ * Version:         Version 1.0
+ * Course:          Advanced Java Programming, Summer 2018
+ * Copyright:       (C) 2018 Michael A. Brooks
+ */
+
+package edu.mbrooks.advancedjava.interfaces;
+
+/**
+ *
+ * @author michaelbrooks
+ * This is a class to setup the basics of what a general collection informaiton is should have
+ */
+public interface Icollectable {
```

Comment:
Don't prefix interfaces with I. Interface or class should be and typically is easily conferred by context. Also make the code harder to read.